bin/git-fork

```diff
@@ -34,10 +34,15 @@ fi
 curl -X POST -u "$user" "https://api.github.com/repos/$owner/$project/forks"
 [ $? = 0 ] || abort "fork failed"
 
-# clone forked repo into current dir
-git clone "https://github.com/$user/$project" "$project"
+if [ "$origin" = true ]; then
+    git remote rename origin upstream
+    git remote add origin "https://github.com/$user/$project"
```

Comment:
Isn't `git fetch origin` enough?

I have several remote refs and some of them are accessible only in my office. I'm afraid that `git fetch --all` causes some kind of network error in my home.